src/utils/kmlUtils.js

```diff
@@ -413,6 +402,15 @@ export function getEditableFeatureFromKmlFeature(kmlFeature, availableIconSets)
 
     const iconStyle = getIconStyle(style)
     const iconArgs = iconStyle ? parseIconUrl(iconStyle.getSrc()) : null
+    if (iconArgs?.isLegacy) {
+        // On the legacy drawing, openlayer used the scale from xml as is, but since openlayer
+        // version 6.7, the scale has been normalized to 32 pixels, therefore we need to add the
+        // 32 pixel scale factor below
+        // scale_factor := iconStyle.getSize()[0] / 32
+        // iconStyle.getSize()[0] = 48 (always 48 pixel on old viewer)
+        // scale_factor = 48/32 => 1.5
+        iconStyle.setScale(iconStyle.getScale() * 1.5)
```

Comment:
it might be good to move the constant `ICON_XML_SCALE_FACTOR` declared in E2E test here, export it, and re-use it in the test too.

Reply:
good point